### ISSUE TITLE
Content updates across app

### ISSUE THREAD

**Issue by nickbrandt** [mozilla-lockbox/lockbox-ios#305]
Some updates to content across the app now that the content team has had the opportunity to do their review.

#### Settings > Account
**InVision**
Account: https://mozilla.invisionapp.com/share/7JEUG47ZU#/268019832_Settings_-_Account
Account - Unlink Confirmation: https://mozilla.invisionapp.com/share/7JEUG47ZU#/268020745_Settings_-_Account_-_Unlink
**Zeplin**
Account: https://zpl.io/a7ZMgyE
Account - Unlink Confirmation: https://zpl.io/2GzN1j9

- [x] Body copy: “To make changes to your account, log in to your Firefox Account from any browser.”
- [x] Unlink account button: "Clear out Lockbox"
- [x] Unlink account button description: "This will remove any synced entries from Lockbox, but will not delete your account or entries from Firefox."
- [x] Unlink account dialog/modal:
Title: "Clear out Lockbox?"
Subtitle: "This will delete your Firefox Account information and all saved entries from Lockbox."

<br/>

#### Legal
- ~Any instances where we have Lockbox, change to Firefox Lockbox~ moving to separate issue

<br/>

#### Auto Lock
**InVision**
https://mozilla.invisionapp.com/share/7JEUG47ZU#/267560622_Settings_-_Auto_Lock
**Zeplin**
https://zpl.io/bAGeO9o

- [x] Table heading: “Sign out of Lockbox after”

<br/>

#### Entry Detail
**InVision**
Entry Detail: https://mozilla.invisionapp.com/share/7JEUG47ZU#/267560603_Entry_Detail
Entry Detail (Tutorial Username): https://mozilla.invisionapp.com/share/7JEUG47ZU#/268014045_Entry_Detail_-_Onboarding_-_Username
Entry Detail (Tutorial Password): https://mozilla.invisionapp.com/share/7JEUG47ZU#/268014046_Entry_Detail_-_Onboarding_-_Password
**Zeplin**
Entry Detail: https://zpl.io/VK4vXDX
Entry Detail (Tutorial Username): https://zpl.io/an1Ejyn
Entry Detail (Tutorial Password): https://zpl.io/adAAYd5

- [x] Copy dialog/modal for Username: “Username copied”
- [x] Copy dialog/modal for Password: “Password copied”
- [x] Edit disclaimer at bottom: “Learn how to edit this entry [chevron-right]”

<br/>

@devinreams moved biometrics to https://github.com/mozilla-lockbox/lockbox-ios/issues/321

**Comment by devinreams** [comment]
>Auto Lock Table heading: “Sign out of Lockbox after”

I'm sure you noticed but two settings are not "times" so this may read confusingly..

- Sign out of Lockbox after 1 minute ✅
- Sign out of Lockbox after Never ⚠️ 
- Sign out of Lockbox after On App exit ⚠️

**Comment by nickbrandt** [comment]
@devinreams good point, this probably makes sense to push back on

**Comment by nickbrandt** [comment]
@devinreams We are working on those changes currently...I have some questions regarding this as it really mucks up a lot of the content if we always have to say "Firefox Lockbox". Ryan got a little antsy on unblocking this issue. Will let you know once we have this fleshed out.

**Comment by devinreams** [comment]
@nickbrandt Understood, I suggest the "renaming to Firefox Lockbox" spin off to a separate issue and branch then as it does get a little funky.

**Comment by nickbrandt** [comment]
Agreed. Will document all of those changes in a separate issue. Thanks for your help with all the content updates!

**Comment by devinreams** [comment]
>Will document all of those changes in a separate issue.

@nickbrandt Cool, thanks.

I also moved the biometrics to #321 since there is some in progress work.

As it stands, this issue and it's checkboxes should now be satisfied with the changes at: #320
